lib/google/ads/google_ads/service_lookup.rb

```diff
@@ -80,17 +80,22 @@ def validate_login_customer_id
           begin
             gax_error.status_details.each do |detail|
               # If there is an underlying GoogleAdsFailure, throw that one.
-              if detail.is_a?(
-                  Google::Ads::GoogleAds::V1::Errors::GoogleAdsFailure)
+              if [
+                Google::Ads::GoogleAds::V1::Errors::GoogleAdsFailure === detail,
```

Comment:
We have a list of known versions. Can we soft-code this so we don't have to update it every time a new version is added or removed?

Also, even if we wanted to keep it this way, is there some advantage to using this wonky syntax over a normal `||` comparison?